scripts/llnl/common_build_functions.py

```diff
@@ -271,7 +271,7 @@ def build_and_test_host_config(test_root, host_config, report_to_stdout=False, e
     bld_output_file =  pjoin(build_dir,"output.log.make.txt")
     print("[starting build]")
     print("[log file: %s]" % bld_output_file)
-    res = sexe("cd %s && make -j 8 VERBOSE=1 " % build_dir,
+    res = sexe("cd %s && make -j 6 VERBOSE=1 " % build_dir,
```

Comment:
This should probably be pulled out to a command line variable and default to something sensible otherwise. I'll make an issue and we can handle it later.

Reply:
#1107